microsoft-edge/visual-studio-code/webhint.md

```diff
@@ -0,0 +1,84 @@
+---
+description: How to use webhint in Visual Studio Code
+title: webhint VS Code extension
+author: hxlnt
+ms.author: raweil
+ms.date: 03/26/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, vs code, visual studio code, webhint
+---
+
+# webhint VS Code extension
+
+[webhint](https://webhint.io) is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors. This open-source project, initially developed by the Microsoft Edge team, is now part of the [OpenJS Foundation](https://openjsf.org/). The Microsoft Edge team continues to contribute to webhint alongside web developers in the community.
```

Comment:
"[webhint](https://webhint.io) is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors."
to
"Use [webhint](https://webhint.io), a customizable linting tool, to improve the accessibility, speed, cross-browser compatibility of your site. It checks your code for best practices and common errors."

Reply:
Fixed (added missing "and" from recommendation).